src/components/BookList.js

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable react/prop-types */
```

Comment:
- [ ] Same here. I would recommend using `prop-types` library here instead of disabling the linters. PropTypes exports a range of validators that can be used to make sure the data you receive is valid. You can check [here](https://www.npmjs.com/package/prop-types) as well.